data/operators.csv

```diff
@@ -60,6 +61,7 @@ DNA Oyj,ISP,,unsafe,16086,523
 Elisa Finland,ISP,,unsafe,719,577
 Reliance Jio,ISP,,unsafe,55836,584
 Spectrum,ISP,,unsafe,12271,607
+Taiwan Fixed Network,ISP,,unsafe,9924,613
```

Comment:
As tested, the status should be safe. Seem like they are filtering as well.

Reply:
No, still the same for me.